inference-engine/src/mkldnn_plugin/config.h

```diff
@@ -41,6 +41,7 @@ struct Config {
 #ifdef CPU_DEBUG_CAPS
     DebugCaps::Config debugCaps;
 #endif
+    std::string cache_dir{};
```

Comment:
why do we need `cache_dir`?

Reply:
Snippets don't support model caching yet, so we need `cache_dir` to know whether the caching is enabled, so we'll disable snippets in the plugin. https://github.com/IvanNovoselov/openvino/blob/bbaf75308ed0df0a93152687442237d455baf4e0/inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp#L552-L557

Reply:
Ok, I can replace string cache_dir Config field with bool modelCacheIsEnabled and assign !val.empty() to it in [readProperties](https://github.com/IvanNovoselov/openvino/blob/8f05c83e2e286dca5207e32e2d214d8332e3666a/inference-engine/src/mkldnn_plugin/config.cpp#L118-L119), but I won't be able to store it in _config in [updateProperties()](https://github.com/IvanNovoselov/openvino/blob/8f05c83e2e286dca5207e32e2d214d8332e3666a/inference-engine/src/mkldnn_plugin/config.cpp#L172).

So to be consistent, I have to introduce a new bool config key PluginConfigParams::KEY_MODEL_CACHING_IS_ENABLED, and implement a mechanism to pass it from ie_core.cpp similar to [the one used for KEY_CACHE_DIR](https://github.com/openvinotoolkit/openvino/blob/e64f94ed7ecb07a7635ec4ccccbf5e4be75e3736/src/inference/src/ie_core.cpp#L777-L782).
This is all doable of cause, but this is s temporary WA (since model caching will be supported by snippets in the future), so I'm not sure it's worth creating a special config key for it.